src/main.ts

```diff
@@ -32,7 +33,7 @@ export function run(args: minimist.ParsedArgs, opts?: TRunOptions) {
       finalOutputDir = path.resolve(finalOutputDir, args.outputDir);
     }
   }
-  let fileSuffix = '.generated.test';
+  let fileSuffix = FILE_SUFFIX;
```

Comment:
please keep the default and if specified, override with the `process.env.value` and finally any command line arguments have priority.
 eg:
```js
let fileSuffix = '.generated.test';
if (process.env.JEST_TEST_GEN_FILESUFFIX) {
    fileSuffix = process.env.JEST_TEST_GEN_FILESUFFIX;
 }
if (args.fileSuffix) {
    fileSuffix = args.fileSuffix;
 }

```